test_scripts/Polices/External_UCS/001_ATF_P_TC_LPT_Creation_PreloadedPT_without_external_consent_status_groups_struct.lua

```diff
@@ -0,0 +1,96 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT without "external_consent_status_groups" struct
+--
+-- Description:
+-- In case:
+-- SDL uploads PreloadedPolicyTable without "external_consent_status_groups:
+-- [<functional_grouping>: <Boolean>]" -> of "device_data" -> "<device identifier>"
+-- -> "user_consent_records" -> "<app id>" section
+-- SDL must:
+-- a. consider this PreloadedPT as valid (with the pre-conditions of all other valid PreloadedPT content)
+-- b. continue working as assigned.
+--
+-- Preconditions:
+-- 1. Stop SDL
+-- 2. Modify PreloadedPolicyTable (remove 'external_consent_status_groups' section)
+--
+-- Steps:
+-- 1. Start SDL
+-- 2. Check SDL status
+--
+-- Expected result:
+-- Status = 0 (SDL is stopped)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+  config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+  config.defaultProtocolVersion = 2
+
+--[[ Required Shared Libraries ]]
+  local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+  local commonSteps = require('user_modules/shared_testcases/commonSteps')
+  local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+  local sdl = require('SDL')
+  local testCasesForExternalUCS = require('user_modules/shared_testcases/testCasesForExternalUCS')
+
+--[[ General Precondition before ATF start ]]
+  commonFunctions:SDLForceStop()
+  commonSteps:DeleteLogsFileAndPolicyTable()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+
+--[[ General Settings for configuration ]]
+  Test = require("user_modules/connecttest_resumption")
+  require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+  commonFunctions:newTestCasesGroup("Preconditions")
+
+  function Test:CheckSDLStatus()
+    testCasesForExternalUCS.checkSDLStatus(self, sdl.RUNNING, "SDL is not running")
+  end
+
+  function Test:StopSDL()
```

Comment:
@istoimenova, The idea here was to make sure SDL works correctly before modification of PreloadedPT